website/templates/public_files.mako

```diff
@@ -0,0 +1,55 @@
+<%inherit file="base.mako"/>
+<%def name="title()"> ${node['ownerName']}'s Public Files</%def>
+
+<%def name="og_description()">
+    Hosted on the Open Science Framework
+</%def>
+
+## To change the postion of alert on project pages, override alert()
+<%def name="alert()"> </%def>
+
+<%def name="content()">
+<div class="page-header  visible-xs">
+</div>
+    <h2 class="text-center"> ${node['ownerName']}'s Public Files</h2>
+
+<div id="treeGrid">
+	<div class="spinner-loading-wrapper">
+		<div class="logo-spin logo-lg"></div>
+		<p class="m-t-sm fg-load-message"> Loading files...  </p>
+	</div>
+</div>
+
+</%def>
+
+<%def name="javascript_bottom()">
+
+<script src="/static/vendor/citeproc-js/xmldom.js"></script>
+<script src="/static/vendor/citeproc-js/citeproc.js"></script>
```

Comment:
This might end up being obvious during functional review, but why/does this page need `citeproc`, `MathJax`, etc?